ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-release-4.15.yaml

```diff
@@ -338,7 +338,56 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: openshift-qe-ocp-qe-perfscale-ci-tests
+    workflow: openshift-qe-cluster-density-v2
+  timeout: 5h0m0s
+- always_run: false
+  as: qe-perfscale-aws-ovn-small-cluster-density
```

Comment:
@jtaleric Can we rename this new test to specify number of nodes instead of "small"? 
1. It will be lot easier to grasp that without looking at details.
2. Will be consistent with our other tests.